sklearn/linear_model/tests/test_base.py

```diff
@@ -156,7 +182,9 @@ def test_linear_regression_sparse(random_state=0):
 
 @pytest.mark.parametrize('normalize', [True, False])
 @pytest.mark.parametrize('fit_intercept', [True, False])
-def test_linear_regression_sparse_equal_dense(normalize, fit_intercept):
+@pytest.mark.parametrize('solver', ['lsqr'])
```

Comment:
FYI, this test fails if I run it with `solver = "cholesky"`. I think this is to be expected, because sparse and dense matrices are treated totally differently by the solver, but wanted to note that here in case I'm understanding something wrong.